pkg/operation/botanist/component/kubeapiserver/kube_apiserver.go

```diff
@@ -161,6 +161,8 @@ type Values struct {
 	StaticTokenKubeconfigEnabled *bool
 	// Version is the Kubernetes version for the kube-apiserver.
 	Version *semver.Version
+	// SeedVersion is the Kubernetes version of the seed cluster.
+	SeedVersion *semver.Version
```

Comment:
No need to introduce this here, or? You can call `Version()` on the `client` you pass to `New`